tests/openvino/native/quantization/test_weights_compression.py

```diff
@@ -80,7 +80,7 @@
 class LMLinearModel(OVReferenceModel):
     HIDDEN_DIM = 16
     OUTPUT_DIM = 32
-    INPUT_SHAPE = [24, HIDDEN_DIM]  # [SeqLen, HiddenDim]
+    INPUT_SHAPE = [1, 24, HIDDEN_DIM]  # [B, SeqLen, HiddenDim]
```

Comment:
Here an below I've updated shapes by adding a batch dimension because current statistics collection framework relies on the assumption that activations have shape [B, L, D].